Makefile

```diff
@@ -14,18 +15,12 @@ test.integration: image
 	make -C example test.arnavon
 
 image:
-	docker build -t quadrabee/arnavon:latest .
+	docker build -t quadrabee/arnavon:${DOCKER_TAG} .
 
-push:
-	docker push quadrabee/arnavon:${VERSION}
+image.push:
+	docker push quadrabee/arnavon:${DOCKER_TAG}
+	docker tag quadrabee/arnavon:${DOCKER_TAG} quadrabee/arnavon:latest
```

Comment:
I'd rather avoid tag builds overriding the "latest" image. Only master is "latest". A tag could be built a-posteriori